packages/loader/driver-utils/src/documentStorageServiceProxy.ts

```diff
@@ -44,6 +47,17 @@ export class DocumentStorageServiceProxy implements IDocumentStorageService {
 		return this.internalStorageService.getSnapshotTree(version, scenarioName);
 	}
 
+	public async getSnapshot(
+		version?: IVersion,
+		snapshotFetchOptions?: ISnapshotFetchOptions,
+	): Promise<ISnapshot | undefined> {
+		assert(
+			this.internalStorageService.getSnapshot !== undefined,
+			"getSnapshot api should exist on documentStorageServiceProxy class",
```

Comment:
I do not know if asserts like that (in couple places) are that helpful. If it's undefined, an exception will be thrown either way. Assert gives us better message (and grouping), but also increases bundle size. If we think it's very unlikely we will hit it, then maybe we should remove and optimize for bundle size.

Reply:
Removed.